code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1800,6 +1800,17 @@
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 	addiction_types = null
 
+/datum/reagent/fuel/oil/expose_turf(turf/exposed_turf, reac_volume)
+	. = ..()
+	if(!istype(exposed_turf) || isspaceturf(exposed_turf)) // Is the turf valid
```

Comment:
what about openspace turfs

Reply:
I am pretty sure it isn't possible to drop in open space.